src/components/text/text.module.css

```diff
@@ -71,73 +69,62 @@
 
 .bodyXl {
   font-size: 2.5rem;
-  font-weight: 400;
+  font-weight: 300;
   line-height: 120%;
   letter-spacing: 0.025rem;
 }
 
 .bodyBig {
   font-size: 1.5rem;
   font-style: normal;
-  font-weight: 400;
-  line-height: 130%;
+  font-weight: 300;
+  line-height: 133.333%;
 }
 
 .bodyNormal {
   font-size: 1.25rem;
   font-style: normal;
   font-weight: 300;
+  line-height: 160%;
+  letter-spacing: 0.0125rem;
 }
 
 .bodySmall {
   font-size: 1rem;
-  font-weight: 300;
+  font-weight: 400;
   line-height: 150%;
 }
 
 .bodyExtraSmall {
-  font-size: 14px;
+  font-size: 0.875rem;
   font-weight: 400;
-  line-height: 18px;
-  letter-spacing: 0.14px;
+  line-height: 128.571%;
+  letter-spacing: 0.00875rem;
 
   @media (max-width: 375px) {
-    font-size: 12px;
-    letter-spacing: 0.12px;
+    font-size: 0.75px;
+    letter-spacing: 0.0075rem;
   }
 }
 
 .labelSmall {
   font-size: 0.875rem;
   font-weight: 400;
-  line-height: 1.5rem;
-}
-
-.labelLight {
-  font-size: 1rem;
-  font-style: normal;
-  font-weight: 300;
-  line-height: 120%;
+  line-height: 171.429%;
 }
 
 .labelRegular {
   font-size: 1rem;
   font-style: normal;
   font-weight: 400;
-  line-height: 120%;
-}
-
-.labelSemibold {
-  font-size: 1rem;
-  font-style: normal;
-  font-weight: 600;
-  line-height: 120%;
+  line-height: 150%;
+  letter-spacing: 0.02rem;
```

Comment:
It's fairly untraditional to add this specifically I think, but it doesn't do any harm if the design actually think that this improves readability etc.

Reply:
Well, I just uncritically added it because it was there 😅